airflow/models/dag.py

```diff
@@ -2020,6 +2025,15 @@ def create_dagrun(
                 "Creating DagRun needs either `run_id` or both `run_type` and `execution_date`"
             )
 
+        try:
+            for k, v in copy.deepcopy(self.params).items():
+                if isinstance(v, Param):
+                    if conf and k in conf:
+                        v.default = conf[k]
+                    v()
+        except ValueError as ve:
+            raise ValueError(f"Invalid input for param '{k}': {ve}")
```

Comment:
```suggestion
            raise ValueError(f"Invalid input for param '{k}': {ve}") from None
```

Otherwise we'll see two stack traces here which is likely confusing and doesn't help our users.